producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBOne.java

```diff
@@ -0,0 +1,171 @@
+                                                                                     "mcolClaim",
+                                                                                     "Rejected",
+                                                                                     "8",
+                                                                                     "First defendant's postcode is not in England or Wales.",
+                                                                                     ""),
+                                                                  Lists.newArrayList("USER_REQUEST_ID_B3",
+                                                                                     "mcolJudgment",
+                                                                                     "Initially Accepted",
+                                                                                     "", "",
+                                                                                     """
+                                                                                          <![CDATA[
+                                                                                                                   <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
+                                                                                                                      <mresp:firstPaymentDate>2014-02-14</mresp:firstPaymentDate>
+                                                                                                                   </ind:mcolResponseDetail>
+                                                                                                                 ]]>
+                                                                                      """),
+                                                                  Lists.newArrayList("USER_REQUEST_ID_B4",
+                                                                                     "mcolClaimStatusUpdate",
+                                                                                     "Initially Accepted",
+                                                                                     "DUP_CUST_REQID", "", "", ""),
+                                                                                     Lists.newArrayList("USER_REQUEST_ID_B5",
+                                                                                     "mcolJudgment",
+                                                                                     "Rejected",
+                                                                                     "24",
+                                                                                     "This judgment request is invalid on the referenced claim.", ""),
```

Comment:
Put the "" parameter on it's own line.

Same comment applies to most of the newArrayList sections below.

Reply:
done